WeakAuras/Animations.lua

```diff
@@ -382,7 +382,9 @@ function Private.Animate(namespace, uid, type, anim, region, inverse, onFinished
       last_update = GetTime()
       updatingAnimations = true;
     end
-    RunAnimation(key, animation, 0, GetTime())
+    C_Timer.After(0, function()
+      RunAnimation(key, animation, 0, GetTime())
+    end)
```

Comment:
lovely

Reply:
Yes indeed... No way that'll create problems in the future.

Reply:
Actually I have an idea...

Reply:
Nah, I won't tinkering with it for now.